apps/web/app/api/epub/route.ts

```diff
@@ -0,0 +1,65 @@
+import { inArray } from "drizzle-orm";
+import epub, { Chapter } from "epub-gen-memory";
+
+import { db } from "@hoarder/db";
+import { bookmarkLinks } from "@hoarder/db/schema";
+
+export async function GET(request: Request) {
+  const { searchParams } = new URL(request.url);
+  const assetIds = searchParams.getAll("assetId");
+
+  if (!assetIds || assetIds.length === 0) {
+    return new Response("", {
+      status: 404,
+    });
+  }
+
+  const bookmarkInformation = await db
+    .select({
```

Comment:
For simple queries like this, prefer `db.query.bookmarkLinks`

Reply:
I guess with the join this is obsolete?